GitOut/App.xaml.cs

```diff
@@ -41,6 +41,7 @@ public App()
 
             ILogger<App> logger = host.Services.GetRequiredService<ILogger<App>>();
             RegisterExceptionHandlers(logger);
+            _ = host.Services.GetService<Features.Wpf.Commands.Application>();
```

Comment:
What is this line doing? `GetService` shouldn't have any side-effects, unless this is somehow creating the Application object?

Reply:
it is creating the application object :)